js/foundation.dropdown.js

```diff
@@ -157,6 +157,17 @@
    */
   Dropdown.prototype._setPosition = function(){
     if(this.$anchor.attr('aria-expanded') === 'false'){ return false; }
+
+    if(this.classChanged){
+      var curPositionClass = this.getPositionClass();
```

Comment:
Copied reset class code from close function. Was on the fence about creating a helper method for both. Zurbs call.